tests/testthat/test-unnecessary_lambda_linter.R

```diff
@@ -80,6 +80,11 @@ test_that("purrr-style anonymous functions are also caught", {
     rex::rex("Pass foo directly as a symbol to map_int()"),
     unnecessary_lambda_linter()
   )
+  expect_lint(
+    "purrr::map_vec(x, ~foo(.x, y))",
+    rex::rex("Pass foo directly as a symbol to map_vec()"),
+    unnecessary_lambda_linter()
```

Comment:
@MichaelChirico is this recommending you use `map_vec(x, foo, y = y)` instead?

We actually recommend using the anonymous function form rather than using `...` to pass named arguments through to `foo()` now. Using the anonymous function form makes it a little easier to see which arguments belong to which function (i.e. does `y` belong to `map_vec()` or `foo()`?), and tends to give slightly better error messages. See the end of the new documentation of `.f` at https://purrr.tidyverse.org/reference/map.html for more.

Reply:
Thanks Davis! Given that recommendation I think we should exclude purrr mappers by default; filed #1871 to handle that before release.